site/src/examples/wealth-and-health-of-nations/index.css

```diff
@@ -0,0 +1,41 @@
+#wealth-and-health-of-nations {
+  margin: 20px auto;
+  width: 100%;
+  height: 400px;
+}
+#wealth-and-health-of-nations > .cartesian-chart,
+.plot-area-container > .plot-area {
+  overflow: visible;
```

Comment:
Seems like these styles are still required, as `svg:not(:root) { overflow: hidden; }` needs to be overridden for `.cartesian-chart`, and I want to override the default overflow (`hidden`) of `.plot-area`.